examples/docker-asyncio/README.md

```diff
@@ -0,0 +1,194 @@
+# docker-asyncio
+
+This example demonstrates an example service running the prio pipeline in docker.
+
+The servers do not handle the public-key exchange, nor does it handle the timing of publishing aggregates. 
```

Comment:
As I stated in conversation, I think public key specification should probably be a configuration step rather than part of the communication protocol. Unless there's something I'm missing?